libr/anal/sign.c

```diff
@@ -343,7 +343,7 @@ R_API bool r_sign_deserialize(RAnal *a, RSignItem *it, const char *k, const char
 			if (token[0] != 0) {
 				it->hash = R_NEW0 (RSignHash);
 				if (it->hash) {
-					it->hash->bbhash = r_str_new (token);
+					it->hash->bbhash = strdup (token);
```

Comment:
thats not the same, r_str_new checks for null and returns null. so the behaviour may differ because the code expects to have bbhash=NULL

```
R_API char *r_str_new(const char *str) {
        return str? strdup (str): NULL;
}
```